### PR TITLE
Add rotation and distortsion filters

### PR DESCRIPTION
**Pull request by Mandruyd**
Added the distortion effect from lavadsp.
Added the new effect added in the latest lavadsp version, rotation.

### CODE REVIEW

File: IMPLEMENTATION.md [hunk 1]
```diff
@@ -176,7 +176,25 @@ JSON comments are for illustration purposes only, and will not be accepted by th
     vibrato: {
         frequency: 2.0, // 0 < x ≤ 14
         depth: 0.5      // 0 < x ≤ 1
+    },
+    // Rotates the sound around the stereo channels/user headphones aka Audio Panning. It can produce an effect similar to: https://youtu.be/QB9EB8mTKcc (without the reverb)
+    rotation: {
+        rotationHz: 0 // the best value would be 0.2, It is he frequency the audio should rotate around the listener, in Hertz.
```
Comment (freyacodes) [inline IMPLEMENTATION.md]: What does “best” mean in this context?

```suggestion
        rotationHz: 0 // The best value would be 0.2, It is he frequency the audio should rotate around the listener, in hertz.
```

Comment (Mandruyd) [reply IMPLEMENTATION.md]: In this context "best" means that the default given value there sounds as similar as to the youtube example there, without the reverb of course.

Comment (freyacodes) [reply IMPLEMENTATION.md]: So “best” is totally subjective? That seems misleading

Comment (freyacodes) [reply IMPLEMENTATION.md]:
```suggestion
        rotationHz: 0 // The frequency of the audio rotating around the listener in Hz. 0.2 is similar to the example video above.
```

**Review by freyacodes — APPROVED** [review]

**Comment by Muh9049** [comment]
Rotation works for me, however, the distortion filter doesn't work. Unless I'm doing something wrong?

**Comment by freyacodes** [comment]
Please post relevant logs

**Comment by Muh9049** [comment]
image

This is what it logs. However, nothing actually ends up being applied as a filter.

**Comment by freyacodes** [comment]
I believe that is the functional equivalent of setting your volume to 100%

**Comment by Muh9049** [comment]
I've tried a couple of different things with different adjustments for sinOffset, sinScale, etc. but the effect just doesn't get applied and the audio stays the same. Someone else in the JDA server said it doesn't seem to work for them either

Also just to double-check, I also lowered the volume of my music to 5% and then tried using my distortion command, however, nothing happens

**Pull request merged**